packages/xflow/package.json

```diff
@@ -82,7 +82,7 @@
     "@ant-design/icons": "^4.6.0",
     "antd": "^4.6.3",
     "lodash": "^4.17.20",
-    "react": "^16.8.0  || ^17.0.0",
-    "react-dom": "^16.8.0  || ^17.0.0"
+    "react": "^16.8.0  || ^17.0.0 || ^18.0.0",
+    "react-dom": "^16.8.0  || ^17.0.0 || ^18.0.0"
```

Comment:
>=16.8.4 ?

Reply:
It may be too loose, It probably has some compatibility problem if react team releases a new version of more than 18.x